aten/src/ATen/native/native_functions.yaml

```diff
@@ -2068,8 +2068,8 @@
     SparseCPU: hspmm_sparse_cpu
     SparseCUDA: hspmm_sparse_cuda
 
-# This "raw copy" doesn't handle conversions NOR does it handle non-blocking.
-- func: raw_copy_sparse_(Tensor self, Tensor src) -> Tensor
+- func: copy_sparse_to_sparse_(Tensor self, Tensor src, bool non_blocking=false) -> Tensor
```

Comment:
Is this function exposed in python? If so, could you blacklist it in `gen_python_functions.py`?

Reply:
right, it is not exposed to python, I will add it to blacklist at `gen_python_functions.py `